### PR TITLE
Re-adds Baneposting sec masks.

### PR DESCRIPTION
**Pull request by ghost**
Bane?
:cl:
add: Sec masks now have a big guy synthesizer built in. Emag them or wirecut them to enable banepost voice lines. Credits to doobiedoo23 for the original code and sounds.
/:cl:

It's an old hippie PR. It deserves to be back in.

### CODE REVIEW

**Comment by myontasm** [comment]
Uh, you don't get to pull code

**Comment by myontasm** [comment]
If I merge that pull will you die?

File: config/admins.txt [hunk 1]
```diff
@@ -130,3 +130,4 @@ JStheguy = Game Master
 excessiveuseofcobby = Game Master
 Plizzard = Game Master
 octareenroon91 = Game Master
+McBawbaggings = Game Master
```
Comment (Carbonhell) [inline config/admins.txt:133]: abuuuse

Comment (ghost) [reply config/admins.txt:133]: fuck you rigatoni stop showing everyone my attempts to subvert the server
but for real, i hate having a file i can't add, it's either that or update the .gitignore to ignore the config folder.

**Review by j-mie — APPROVED** [review]

**Pull request merged**